tests/integration/test_records.py

```diff
@@ -351,19 +353,78 @@ def test_get_es_records_accepts_lists_of_strings(app):
 
 
 def test_records_files_attached_correctly(app):
-    json = {
+    record_json = {
         '$schema': 'http://localhost:5000/schemas/records/hep.json',
         'document_type': [
             'article',
         ],
         'titles': [
             {'title': 'foo'},
         ],
-        '_collections': ['Literature']
+        '_collections': [
+            'Literature'
+        ]
     }
 
-    record = InspireRecord.create(json)
+    record = InspireRecord.create(record_json)
     record.files['fulltext.pdf'] = StringIO.StringIO()
     record.commit()
 
     assert 'fulltext.pdf' in record.files
+
+
+@patch('inspirehep.modules.records.utils.open', mock_open())
+def test_records_files_are_populated_correctly_for_local_paths(app):
```

Comment:
this probably should be renamed to `test_attach_documents_and_figures_....`, it's kind of a convention to start the (specially unit) tests with `test` followed by the name of the function or class it tests.